tendermint-rs/src/block/height.rs

```diff
@@ -9,7 +9,7 @@ use std::{
 /// Block height for a particular chain (i.e. number of blocks created since
 /// the chain began)
 #[derive(Copy, Clone, Eq, Hash, PartialEq, PartialOrd, Ord)]
-pub struct Height(pub u64);
+pub struct Height(u64);
```

Comment:
With `pub` on the `u64` this wraps, the `Height` type can't maintain invariants (i.e. `>0` or `>=0`)